src/libostree/ostree-repo-libarchive.c

```diff
@@ -631,6 +635,7 @@ aic_handle_file (OstreeRepoArchiveImportContext *ctx,
                  GCancellable       *cancellable,
                  GError            **error)
 {
+  GLNX_AUTO_PREFIX_ERROR ("ostree-tar: Failed to import file", error);
```

Comment:
```suggestion
  GLNX_AUTO_PREFIX_ERROR ("ostree-tar: Failed to handle file", error);
```

So that it's different from the prefix for `aic_import_file` (and more closely matches the function name).

Would be good to include the path in there too, though ideally just in a few places so it's not repeated all the way up.